site/en/blog/new-in-chrome-96/index.md

```diff
@@ -0,0 +1,222 @@
+url as the unique id, should land in the first half of 2022.
+
+{% Img class="float-right", linkTo="true", src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/XY3FVUeeCjhB9K71niJE.png", alt="DevTools showing computed app id", width="800", height="533" %}
+
+If you already have a PWA in production and want to add an `id` to your
+manifest, you'll need to use the id that was assigned by the browser. You can
+find the `id` in the _Manifest_ pane of the _Application_ panel in Dev Tools.
+
+<p style="clear:both;"></p>
+
+For a brand new PWA, you can set the `id` to any string value you like, but
+remember, you won't be able to change it in the future, so choose wisely.
+
+```json/2
+{
+  ...
+  id: "SquooshApp",
+  start_url: "/?homescreen=1",
+  ...
+}
+```
+
+Check out [Uniquely identifying PWAs with the web app manifest id property][manifest-id]
+for more info.
+
+## URL protocol handlers for PWAs {: #pwa-protocol-handlers }
+
+Web apps can use [`navigator.registerProtocolHandler()`][mdn-regph] to
+register as a protocol handler. For example, Gmail can register the `mailto`
+protocol, then when a user clicks on a link with the `mailto:` prefix, Gmail
+will open making it easy for the user to send an email.
+
+Starting in Chrome 96, a PWA can register as protocol handler as part of its
+installation. To do that for your PWA, add a `protocol_handlers` property to
+your web app manifest, specify the `protocol` you want to handle, and the
+`url` that should be opened when clicked.
+
+```json
+  ...
+  "protocol_handlers": [
+    {
+      "protocol": "web+tea",
+      "url": "/tea?type=%s"
+    },
+    {
+      "protocol": "web+coffee",
+      "url": "/coffee?type=%s"
+    }
+  ]
+}
+```
+
+There are some restrictions here, and you can't just register any protocol, so
+check out [URL protocol handler registration for PWAs][wd-phreg] for complete
+details, and how you can use the `web+` syntax to create your own protocols!
+
+## Priority hints (origin trial) {: #pri-hints }
+
+When a browser parses a web page and begins to discover and download resources
+such as images, scripts, or CSS, it assigns them a fetch priority to try to
+optimize the page load. Browsers are pretty good at assigning priorities, but
+may not be optimal in all cases.
+
+Priority Hints are an experimental feature, available as an
+[origin trial][ph-ot] starting in Chrome 96, and can help optimize the Core
+Web Vitals. The `importance` attribute allows you to specify the priority
+for resource types such as CSS, fonts, scripts, images, and iframes.
+
+```html
+<!-- We don't want a high priority for this above-the-fold image -->
+<img src="/not-important.svg" importance="low">
+
+<!-- Initiate an early fetch for a resource, but de-prioritize it -->
+<link rel="preload" href="/script.js" as="script" importance="low">
+
+<script>
+  fetch('https://example.com/', {importance: 'high'}).then(data => {
+    // Trigger a high priority fetch
+  });
```

Comment:
```suggestion
  fetch('https://example.com/', {importance: 'high'})
  .then(data => {
    // Trigger a high priority fetch
  });
```

Reply:
Fixed in 1475b8e031abe034773b744274e64066738153ad